src/components/Popups/LiquidityV3Popup.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components'
+
+import GnoTokenLogoDark from '../../assets/images/gnosis-chain-logo-dark.svg'
+import SwaprTokenLogoDarkWithRing from '../../assets/images/swapr-logo-dark-with-ring.svg'
+import { LIQUIDITY_V3_LINK } from '../../constants'
+
+export const LiquidityV3Popup = () => {
+  return (
+    <FlexContainer onClick={() => document.location.assign(LIQUIDITY_V3_LINK)}>
```

Comment:
Can we define `FlexContainer` as a anchor element `<a>` and use `href` and `target` props to open the link? This should open on a new window, so user flow doesn't get disrupted by clicking a link.

Reply:
Agree, will do!

Reply:
`target="_blank"` to open on new tab